### PR TITLE
[Shader Graph][Skip CI] Bug Fixes

### PR DESCRIPTION
**Pull request by alindmanUnity**
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
Fixes multiple shader graph bugs: 
- Fixed a bug where the `Vertex Normal` slot on master nodes needed vertex normal data input to compile. [1193348](https://issuetracker.unity3d.com/issues/hdrp-unlit-shader-plugging-anything-into-the-vertex-normal-input-causes-shader-to-fail-to-compile)
- Fixed a bug where `GetWorldSpaceNormalizeViewDir()` could cause undeclared indentifier errors. [1190606](https://issuetracker.unity3d.com/issues/view-dir-node-plugged-into-vertex-position-creates-error-undeclared-identifier-getworldspacenormalizeviewdir)
- Fixed a bug where emission on PBR Shader Graphs in Universal would not bake to lightmaps. [1190225](https://issuetracker.unity3d.com/issues/emissive-custom-pbr-shadergraph-material-only-works-for-primitive-unity-objects)
- Fixed a bug where Shader Graph shaders would not build to PS4 writing to `POSITION` semantic. [internal only](https://fogbugz.unity3d.com/f/cases/1160305/)
---
### Testing status

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [x] Built a player
- [x] Checked new UI names with UX convention
- [x] Tested UI multi-edition + Undo/Redo + Prefab overrides + Alignment in Preset
- [x] C# and shader warnings (supress shader cache to see them)
- [x] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 
Tested locally in Universal and HD test projects as well.

**Automated Tests**:  Baked GI for Universal should be added in the Universal test project, I'll talk to Universal team about this. 

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/sg%252Fjan-bug-bash-alex

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Comment by alindmanUnity** [comment]
> Bug 1190606 was fixed for unlit, was not fixed for PBR
@LandonTownsendUnity  can you confirm fix at commit 69956d0

**Comment by LandonTownsendUnity** [comment]
Fix confirmed for PBR on bug 1190606. Will approve as soon as I have word that someone has confirmed the fix for PS4 builds (I am unable to because my license does not permit me to build for PS4)

File: com.unity.shadergraph/Editor/ShaderGUI/PBRMasterGUI.cs [hunk 1]
```diff
@@ -9,17 +9,13 @@ public override void OnGUI(MaterialEditor materialEditor, MaterialProperty[] pro
...
-                if (prop.name == "_EmissionColor")
-                {
-                    if (materialEditor.EmissionEnabledProperty())
-                    {
-                        materialEditor.LightmapEmissionFlagsProperty(MaterialEditor.kMiniTextureFieldLabelIndentLevel, true);
-                    }
-                    return;
-                }
-            }
+                // materialEditor.LightmapEmissionFlagsProperty(MaterialEditor.kMiniTextureFieldLabelIndentLevel, true, true);
```
Comment (phi-lira) [inline com.unity.shadergraph/Editor/ShaderGUI/PBRMasterGUI.cs]: Minor: remove commented code.

**Comment by sambickley** [comment]
ShaderGraph Test Project passes testing on PS4.

**Comment by j-reichman** [comment]
@alindmanUnity 

Hello, I am the author of the 1190225 Bug report (https://issuetracker.unity3d.com/issues/emissive-custom-pbr-shadergraph-material-only-works-for-primitive-unity-objects)

I just wanted to point out that the provided fix doesn't solve the issue. While having the emissive toggle for materials for setting proper lightmap flag is definitely appreciated and helpful, the main issue still persist.

If you open the attached project in the bug report, you can see that the lightmap flag was correctly set  on mateiral via Debug mode in inspector, and the object still wont bake. However, it only doesn't bake on custom made 3D models, Unity's default primitives work for some reason.

Thank you!

**Pull request merged**